packages/utils/src/lambda/execute-lambda.ts

```diff
@@ -58,17 +57,9 @@ async function main() {
       log: logResultToString(lambdaResult.LogResult),
     });
   }
+  const response = getLambdaResultPayload({ result: lambdaResult, lambdaName });
 
-  if (isLambdaError(lambdaResult)) {
-    console.log(
-      `Error calling lambda ${lambdaName}: ${JSON.stringify(getLambdaError(lambdaResult), null, 2)}`
-    );
-    return;
-  }
-
-  const response = JSON.parse(lambdaResult.Payload.toString());
-
-  const url = response.url;
+  const url = JSON.parse(response).url;
```

Comment:
Use `getLambdaResultPayload` instead of building its own version - likely a left-over from when we created that shared function.